spec.md

```diff
@@ -447,10 +447,10 @@ Not all permutations of plan changes are expected to be supported. For example,
 | plan_id | string | The ID of the plan (from the catalog) for which the service instance has been requested. MUST be unique to a service. |
 | parameters | JSON object | Configuration options for the service instance. An opaque object, controller treats this as a blob. Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
 | previous_values | object | Information about the instance prior to the update. |
-| previous\_values.service_id | string | ID of the service for the instance. |
-| previous\_values.plan_id | string | ID of the plan prior to the update. |
-| previous\_values.organization_id | string | ID of the organization specified for the instance. |
-| previous\_values.space_id | string | ID of the space specified for the instance. |
+| previous_values.service_id | string | ID of the service for the instance. |
+| previous_values.plan_id | string | ID of the plan prior to the update. |
+| previous_values.organization_id | string | ID of the organization specified for the instance. |
+| previous_values.space_id | string | ID of the space specified for the instance. |
```

Comment:
I believe the escaped underscores were required to prevent the CF docs from styling characters between them as italic.